golemcli/src/main.rs

```diff
@@ -52,6 +52,12 @@ struct CliArgs {
     #[structopt(long, short)]
     net: Option<Net>,
 
+    /// Answer `YES` to any prompt without an user action
+    #[structopt(short = "y")]
```

Comment:
I'd enable also long option as it is in http://manpages.ubuntu.com/manpages/cosmic/man8/apt-get.8.html
>        -y, --yes, --assume-yes
>           Assume "yes" as an answer to all prompts and run non-interactively.

There's also

>        --assume-no
>           Automatic "no" to all prompts.

Reply:
ok